daemon/cmd/daemon_main.go

```diff
@@ -1016,6 +1016,9 @@ func InitGlobalFlags(cmd *cobra.Command, vp *viper.Viper) {
 	flags.StringSlice(option.VLANBPFBypass, []string{}, "List of explicitly allowed VLAN IDs, '0' id will allow all VLAN IDs")
 	option.BindEnv(vp, option.VLANBPFBypass)
 
+	flags.Bool(option.DisableExternalIPMitigation, false, "Disable ExternalIP mitigation (CVE-2020-8554, default false)")
+	option.BindEnv(vp, option.DisableExternalIPMitigation)
+
```

Comment:
Could you mention this option in https://github.com/cilium/cilium/blob/main/Documentation/network/kubernetes/kubeproxy-free.rst - the DSR section maybe?

Reply:
Will make change and will get back.

Reply:
Fixed.